benchmark/bench_CQRRPT/CQRRPT_runtime_breakdown.cc

```diff
@@ -117,14 +113,105 @@ int main() {
     RandLAPACK::gen::mat_gen(m_info, all_data.A.data(), state);
 
     // Declare a data file
-    std::fstream file("CQRRPT_inner_speed_"              + std::to_string(m)
+    std::string output_filename = "CQRRPT_inner_speed_"              + std::to_string(m)
                                       + "_col_start_"    + std::to_string(n_start)
                                       + "_col_stop_"     + std::to_string(n_stop)
                                       + "_d_factor_"     + std::to_string(d_factor)
-                                      + ".dat", std::fstream::app);
+                                      + ".dat";
 
     for (;n_start <= n_stop; n_start *= 2) {
-        res = call_all_algs(m_info, numruns, n_start, all_data, state_constant);
-        file << res[0]  << ",  " << res[1]  << ",  " << res[2] << ",  " << res[3] << ",  " << res[4] << ",  " << res[5] << ",  " << res[6] << ",  " << res[7] << ",\n";
+        call_all_algs(m_info, numruns, n_start, all_data, state_constant, output_filename);
+    }
+}
+*/
+/*
+int main(int argc, char *argv[]) {
+
+    printf("Function begin\n");
+
+    if(argc <= 1) {
+        printf("No input provided\n");
+        return 0;
     }
-}
\ No newline at end of file
+
+    // Declare parameters
+    int64_t m              = 0;
+    int64_t n              = 0;
+    double  d_factor       = 1.25;
+    double  tol            = std::pow(std::numeric_limits<double>::epsilon(), 0.85);
+    auto    state          = RandBLAS::RNGState();
+    auto    state_constant = state;
+    // Timing results
+    std::vector<long> res;
+    // Number of algorithm runs. We only record best times.
+    int64_t numruns = 20;
+
+    // Generate the input matrix.
+    RandLAPACK::gen::mat_gen_info<double> m_info(m, n, RandLAPACK::gen::custom_input);
+    m_info.filename = argv[1];
+    m_info.workspace_query_mod = 1;
+    // Workspace query;
+    RandLAPACK::gen::mat_gen<double>(m_info, NULL, state);
+    // Update basic params.
+    m = m_info.rows;
+    n = m_info.cols;
+    // Allocate basic workspace.
+    QR_benchmark_data<double> all_data(m, n, tol, d_factor);
+    // Fill the data matrix;
+    RandLAPACK::gen::mat_gen(m_info, all_data.A.data(), state);
+
+    // Declare a data file
+    std::string output_filename = "CQRRPT_inner_speed_" + std::to_string(m)
+                                      + "_col_start_"  + std::to_string(n)
+                                      + "_col_stop_"   + std::to_string(n)
+                                      + "_d_factor_"   + std::to_string(d_factor)
+                                      + ".dat";
+
+    call_all_algs(m_info, numruns, n, all_data, state_constant, output_filename);
+}
+*/
```

Comment:
Everything from 94 to the end of this block (line 172) is commented out. Please remove.

Reply:
Done.